styles/about.css

```diff
@@ -0,0 +1,672 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+  max-width: 100%;
+  overflow-x: hidden;
+}
+
+:root {
+  --red: #ec5242;
+  --black: #272a31;
+  --gray: #d3d3d3;
+}
+
+.bg-top {
+  background-color: #f0f0f0;
+}
+
+.social-media-icons {
+  color: white;
+  font-size: 2rem;
+}
+
+#main-container {
+  min-height: 100vh;
```

Comment:
- [ ] **OPTIONAL**
For the styling of the mobile menu, and as the first reviewer mentioned is it should follow the color/pattern from the top section of the homepage or to have a transparent background. I highly recommend working on this to maintain the website professional.